wp-admin/js/widgets/media-widgets.js

```diff
@@ -73,6 +73,7 @@ wp.mediaWidgets = ( function( $ ) {
 		 * @type {Object}
 		 */
 		events: {
+			'click .notice a': 'selectMedia',
```

Comment:
I think I prefer the former, [I'd like to avoid a situation](https://github.com/xwp/wp-core-media-widgets/pull/27#discussion_r106741038) where translators accidentally break functionality.